Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ConfigurationHistory/ConfigurationHistory.tsx

```diff
@@ -133,7 +132,7 @@ function ConfigurationHistory(props: { MeterConfigurationID: number, MeterKey: s
                         <button className="btn btn-danger pull-right" onClick={getData} disabled={!changed}>Reset</button>
                     </div>
                 </div>
-                <div className={"tab-pane " + (tab == "filesProcessed" ? " active" : "fade")} id="filesProcessed">
+                <div className={"tab-pane " + (Tab == "filesProcessed" ? " active" : "fade")} id="filesProcessed">
                     <div style={{ width: '100%', maxHeight: window.innerHeight - 275, padding: 30, overflowY: 'auto' }}>
```

Comment:
is that cleaned up sepperately? `maxHeight: window.innerHeight - 275` is generally not great would like to switch to flex, row, col etc